zulipterminal/helper.py

```diff
@@ -35,6 +35,7 @@
     'edited_messages': Set[int],  # {message_ids, ...}
     'topics': Dict[int, List[str]],  # {topic names, ...}
     'search': Set[int],  # {message_id, ...}
+    'groups': Dict[int, Dict[str, Any]],
```

Comment:
@neiljp Yes. I added it to the index so that it would be usable in the future if we were to add support for creating / deleting / adding members to a user group. You are right as I have not made use of this content in the index.

Maybe, we could have this later on and for now just go with storing it in the model. `user_group_by_id` sounds good.